torch/utils/tensorboard/_proto_graph.py

```diff
@@ -5,11 +5,7 @@
 
 
 def attr_value_proto(dtype, shape, s):
-    """Creates a dict of objects matching
-    https://github.com/tensorflow/tensorboard/blob/master/tensorboard/compat/proto/attr_value.proto
-    specifically designed for a NodeDef. The values have been
-    reverse engineered from standard TensorBoard logged data.
-    """
+    """Create a dict of objects matching https://github.com/tensorflow/tensorboard/blob/master/tensorboard/compat/proto/attr_value.proto specifically designed for a NodeDef. The values have been reverse engineered from standard TensorBoard logged data."""
```

Comment:
Why does this whole URL https://github.com/tensorflow/tensorboard/blob/master/tensorboard/compat/proto/attr_value.proto needs to be in the first line? Just rephase and put it later.

Reply:
Fixed it